### ISSUE TITLE
Help! Stuck in an empty world?! Tried reinstall, no add-ons... no idea how this happened. :(

### ISSUE THREAD

**Issue by remmurd20** [dalaranwow/dalaran-wow#5513]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
My character Neukemia cannot see any other live characters of anything interactive (mailbox etc.) I can go to different zones, queue for bg's and join, but no one is there and nothing can be interracted with. I have tried no add-ons. I have tried complete re-download, but no fix with this character only. I can even see global chat, and communicate in it, but i have a blank world. I can't even mail my BoAs or money off the character in the meantime... It's been days!

My other characters work perfectly fine...
**Expected behaviour**: Tell us what should happen instead
I should have a normally functioning character that I can play World of Warcraft with.

**Steps to reproduce the problem**:
I have no idea what caused it - none. But I'm still stuck in my own netherworld. Feel free to log onto Neukemia - nothing but an empty world.

**Include proofs for this behaviour**
![wowscrnshot_061418_195523](https://user-images.githubusercontent.com/16918511/41443854-66f1ba94-700d-11e8-8e3e-2a1ea20d7fcf.jpg)
![wowscrnshot_061418_195540](https://user-images.githubusercontent.com/16918511/41443855-67016304-700d-11e8-996c-c78d037b9ec2.jpg)
![wowscrnshot_061418_195548](https://user-images.githubusercontent.com/16918511/41443856-670ee182-700d-11e8-90f1-890e02d87e68.jpg)
![wowscrnshot_061418_195559](https://user-images.githubusercontent.com/16918511/41443857-671c67a8-700d-11e8-8106-34d55fe1cafd.jpg)
![wowscrnshot_061418_195617](https://user-images.githubusercontent.com/16918511/41443858-672b4570-700d-11e8-8c88-0d314a1c78c4.jpg)
![wowscrnshot_061418_195700](https://user-images.githubusercontent.com/16918511/41443859-673879ca-700d-11e8-9827-ab3c3d4545ad.jpg)
![wowscrnshot_061418_195713](https://user-images.githubusercontent.com/16918511/41443860-6747c7cc-700d-11e8-9a05-42d372db9053.jpg)
![wowscrnshot_061418_195800](https://user-images.githubusercontent.com/16918511/41443861-6756ecca-700d-11e8-8fa2-48d2893175fe.jpg)
![wowscrnshot_061418_195823](https://user-images.githubusercontent.com/16918511/41443862-6763b1c6-700d-11e8-97d0-df6064556f85.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
It should be fine already. Should the issue persist or happen to you again, please don't hesitate to take it to the in-game support ticket system and we will help you out.

If the issue still persists:
1) Kill your character on purpose
2) Abandon any quest that could be messing with your phasing
3) Make an in-game ticket

Closing up

**Comment by remmurd20** [comment]
Golganeth - I'll try hopping on now. But I opened a ticket 2 days ago

**Comment by Golganeth** [comment]
I can't see your ticket. Did I fix it or do you still need assistance? Keep in mind that in most cases, if your character is offline we won't be able to help you

If the issue persists please try the steps above.
Cheers
